### DISCUSSION TITLE
Many to many query

### DISCUSSION THREAD

**Discussion by LogicalChaos** [aarondl/sqlboiler#1024] [Q&A] [answered]
I'm having no luck trying to figure out how to query two tables with a many to many relationship.  Take the Video <-> Tag examples with a tag_id/video_id join table.  I would like to use SQLBoiler to find all videos with tags romantic AND comedy.  
Any help for how to do this?

**Comment by aarondl** [comment, marked as answer]
In these cases sqlboiler isn't going to help too much. Generally I'd recommend you simply use a custom query & `queries.Bind` to bind the result back into a slice of videos.

**Reply by LogicalChaos** [reply]
That explains why I couldn't figure it out.  I feel better now, thanks :-)
